.github/workflows/docs-and-linting.yml

```diff
@@ -0,0 +1,49 @@
+name: Render and Lint Documentation
+
+on:
+  pull_request:
+    branches_ignore: []
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.10', '1.11', '1.12']
```

Comment:
these old versions are fine for first pass at parity. We can add the newer golang versions later